front-end/src/app/candidate/data-candidate/data-candidate.component.ts

```diff
@@ -185,11 +184,51 @@ export class DataCandidateComponent implements OnInit {
 
   // load user profile picture
   imageConstruction(){
-    if ( this.dataCandidateForm.get('profileImage').value != null ) {
+    if ( this.candidate.profileImage != null ) {
       let basePath: string = "/api/v1/uploads/";
-      this.imageUrl = environment.apiUrl + basePath + this.dataCandidateForm.get('profileImage').value;
+      this.imageUrl = environment.apiUrl + this.candidate.profileImage;
       this.nameButton = 'Change Photo';
       this.activeRemove = true;
     }
   }
-}
+
+  uploadFile(event: any){
+    let valueId: string = event.target.id;
+    
+    if ( valueId == "resumeUrl"){
+      let textResume = document.getElementById('text-' + valueId);
+      textResume.innerHTML = event.target.files[0].name;
+      this.listfile[1] = event.target.files[0];
```

Comment:
For better readability, I recommend naming this variable `listfile[1]` to something specific as `resumeUrl`.

Reply:
> For better readability, I recommend naming this variable `listfile[1]` to something specific as `resumeUrl`.

Modified